Import blockscopy too and rename it to copyblocks (#3264)
grafana/mimir: tools/copyblocks/README.md

> # Copy blocks

Reviewer: ```suggestion
# Copyblocks
```

This program can copy Mimir blocks between two GCS buckets. It is GCS specific and uses special calls to copy
files between buckets directly, without download to a local system first.

> copyblocks can run in one-time mode, or as a service and do regular copies.

Author: What we mean here is that the program can run a one-time copy or can run continuously (as a service) that periodically check if there are blocks to copy and run the copy.

> Blocks copy can run in one-time mode, or as a service and do regular copies.

Reviewer 2: ```suggestion
copyblocks can run in one-time mode, or as a service and do regular copies.
```

> It can be configured with minimum block time range to avoid copying of blocks that are too small.

Reviewer: ```suggestion
You can configure it with a minimum block time range to avoid copying blocks that are too small.
```

> It can also be configured with list of users to copy, or avoid copying blocks for.

Reviewer: ```suggestion
You can also list users to copy, or avoid copying blocks for.
```
What does, "or avoid copying blocks for" mean? Cannot parse this meaning as-is.

Author: You can configure an allow list and a block list. By default all tenants are copied.
